.github/workflows/publish-image.yml

```
@@ -0,0 +1,43 @@
+name: Publish Image
+on:
+  push:
+    branches:
+      - main
+    tags:
+      - '*'
+    paths:
+      - ./version.json
+  pull_request:
+    branches:
+      - '*'
+
+jobs:
+  build:
+    env:
+      context: .
+      image_name: swoop-db
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@main
+      - name: Get image tags
+        id: image_tags
+        run: |
+          echo -n ::set-output name=IMAGE_TAGS::
+          VERSION=$(jq -r '.version' ${context}/version.json)
+          TAGS=('latest')
+          if [ "${VERSION}" ] && [ "${VERSION}" != "latest" ]; then
+              TAGS+=("${VERSION}")
+          fi
+          if [[ "${GITHUB_REF}" =~ refs/tags/(.*) ]]; then
+              TAGS+=("git-${BASH_REMATCH[1]}")
+          fi
+          ( IFS=$','; echo "${TAGS[*]}" )
```

I am thinking we can probably remove the `version.json` file and use the same mechanism the python release uses to get the package version by doing something like this:

```suggestion
      - name: Set up Python ${{ matrix.python-version }}
        uses: actions/setup-python@v4
        with:
          python-version: '3.x'

      - name: Install dependencies
        run: pip install setuptools-scm

      - name: Get image tags
        id: image_tags
        run: |
          echo -n ::set-output name=IMAGE_TAGS::
          VERSION="$(python -m setuptools_scm)"
          TAGS=('latest' "${VERSION}")
          if [[ "${GITHUB_REF}" =~ refs/tags/(.*) ]]; then
              TAGS+=("git-${BASH_REMATCH[1]}")
          fi
          ( IFS=$','; echo "${TAGS[*]}" )
```